devices/pine64-pinephonepro/kernel/default.nix

```
@@ -0,0 +1,34 @@
+{
+  mobile-nixos
+, fetchpatch
+, fetchFromGitLab
+, ...
+}:
+
+mobile-nixos.kernel-builder {
+  version = "6.0.8";
+  configfile = ./config.aarch64;
+
+  src = fetchFromGitLab {
+    owner = "pine64-org";
+    repo = "linux";
+    rev = "ppp-6.0.8-ppkb";
+    sha256 = "sha256-t7OwU6wRQlU2XCfS1Vd2yWlWzoneFqO0qPa8AZWIAmU=";
+  };
+
+  patches = [
+    ./0001-HACK-always-show-logo.patch
```

```suggestion
    ./0001-HACK-always-show-logo.patch
```

(And remove the file)

I know I added this patch, I'm not sure we should enable it outright in the end. (After all this was a really-WIP branch I published at the time.)

Or if it is desirable, I think this shouldn't be part of a specific device since this is an overall problem, not a Pinephone Pro problem.